packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -180,7 +184,23 @@ class TimelineScreen extends Screen {
       }
     });
 
-    onSelectedFrameFlameChartItem.listen(eventDetails.update);
+    onSelectedFrameFlameChartItem.listen((FrameFlameChartItem item) async {
+      final TimelineEvent event = item.event;
+
+      String gaTimeLineFlame;
+      if (event.isGpuEvent) {
+        gaTimeLineFlame = ga.timelineFlameGpu;
+      } else {
+        gaTimeLineFlame = ga.timelineFlameUi;
+      }
+      ga.select(
+        ga.timeline,
+        gaTimeLineFlame,
```

Comment:
nit: you could simplify and in-line this here:
`event.isUiEvent ? ga.timelineFlameUi : ga.timelineFlameGpu`

Reply:
done.